discovery-server/src/main/java/io/airlift/discovery/store/Entry.java

```diff
@@ -73,7 +73,7 @@ public long getTimestamp()
         return timestamp;
     }
 
-    @JsonProperty
+    @JsonProperty("maxAge")
```

Comment:
Maybe it's better to change the constructor to accept `maxAgeInMs`?

Reply:
Yes. Updated and changed the constructor.